packages/core/src/components/popover/popover.tsx

```diff
@@ -124,6 +124,14 @@ export interface IPopoverProps extends IOverlayableProps, IProps {
     onInteraction?: (nextOpenState: boolean) => void;
 
     /**
+     * Whether the popover should open when its target is focused.
+     * If `true`, target will render with `tabindex="0"` to make it focusable via keyboard navigation.
+     * This prop is only available when `interactionKind` is `HOVER` or `HOVER_TARGET_ONLY`.
```

Comment:
you know what: after a close reading of the code i'm pretty into what you've done here. functionality _is_ mixed into those two interaction kinds, but it can also be flagged off if needed. 👍